EIPS/draft-factory-standard.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: <to be assigned>
+title: Standard factory contract format
+description: A standard format for factory contracts
```

Comment:
Recommend a more descriptive description if possible.  Shouldn't be too long, but usually longer than the title.

Reply:
done in https://github.com/ethereum/EIPs/pull/5202/commits/b36c0bee2cb4ccf4643710b1ae667764a83adedf